Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
I'm not sure it's worth trying to enforce an indent style onto formulae at this point, where those are the only two options. It's semi-reasonable to say that different usages merit different indents & enforcing consistency without more active consideration may actually make things look even stranger.